.github/workflows/code_coverage.yml

```diff
@@ -0,0 +1,32 @@
+name: Code Coverage Report
+
+on:
+  pull_request:
+    branches:
+      - code_coverage_setup
+
+jobs:
+  code-coverage:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Instrumentation
+        run:  npx nyc instrument ./ instrumented
+
+      - name: Move folder
+        run: mv config instrumented
+
+      - name: Run tests with coverage
+        run: npm run test:coverage
```

Comment:
Instead of running the separate command/workflow for test coverage, can't we merge it with `cypress-tests.yml` workflow?